src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const getListMen = people.filter(person => person.sex === 'm', 0);
```

Comment:
You don't have to specify the index (as a second parameter) in filter method if index = 0. Also you can create helper functions to get people by their sex (males, females) and to get average age. Because you will be able to use them in another functions in this task.

Reply:
1. with 0 agree 2. helper function - i thought about, but we have to create several different functions in this task